docs.md

```diff
@@ -589,6 +589,13 @@ swift_name_macro = "CF_SWIFT_NAME"
 # default: "None"
 rename_args = "PascalCase"
 
+# This rule specifies if the order of functions will be sorted in some way.
+#
+# "ByName": sort by the name of the function
+# "Keep": keep order in which the functions have been parsed
+#
+# default: "ByName"
```

Comment:
A bit bikesheddy, but I think this would be a bit better if the variant was just `Name`. At least to me `sort_by = "Name"` reads better than `sort_by = "ByName"`.

Similarly, mostly for consistency with renaming options, probably the "keep" variant should be `None`.

Reply:
You are right. This feels a lot more natural.